databricks/sdk/service/iam.py

```diff
@@ -1175,12 +1179,24 @@ def list(self,
         if sort_order is not None: query['sortOrder'] = sort_order.value
         if start_index is not None: query['startIndex'] = start_index
         headers = {'Accept': 'application/json', }
-        json = self._api.do('GET',
-                            f'/api/2.0/accounts/{self._api.account_id}/scim/v2/Groups',
-                            query=query,
-                            headers=headers)
-        parsed = ListGroupsResponse.from_dict(json).resources
-        return parsed if parsed is not None else []
+
+        # deduplicate items that may have been added during iteration
+        seen = set()
+        query['startIndex'] = 0
```

Comment:
This is incorrect but will be fixed in https://github.com/databricks/databricks-sdk-py/pull/433/files (start index should be 1).